packages/flutter/test/material/paginated_data_table_test.dart

```diff
@@ -1043,4 +1044,81 @@ void main() {
     await tester.pumpWidget(buildFrame(overflowBar));
     expect(headerX, tester.getTopLeft(find.byType(ElevatedButton)).dx);
   });
+
+  testWidgets('PaginatedDataTable can be scrolled using ScrollController', (WidgetTester tester) async {
+    final TestDataSource source = TestDataSource();
+    final ScrollController scrollController = ScrollController();
+
+    Widget buildTable(TestDataSource source) {
+      return Align(
+        alignment: Alignment.topLeft,
+        child: SizedBox(
+          width: 100,
+          child: PaginatedDataTable(
+            controller: scrollController,
+            header: const Text('Test table'),
+            source: source,
+            rowsPerPage: 2,
+            columns: const <DataColumn>[
+              DataColumn(
+                label: Text('Name'),
+                tooltip: 'Name',
+              ),
+              DataColumn(
+                label: Text('Calories'),
+                tooltip: 'Calories',
+                numeric: true,
+              ),
+              DataColumn(
+                label: Text('Generation'),
+                tooltip: 'Generation',
+              ),
+            ],
+          ),
+        ),
+      );
+    }
+
+    await tester.pumpWidget(MaterialApp(
+      home: buildTable(source),
+    ));
+
+    expect(scrollController.offset, 0.0);
+
+    scrollController.jumpTo(50.0);
+    await tester.pumpAndSettle();
+    expect(scrollController.offset, 50.0);
```

Comment:
This does not actually verify which scrollable used the scroll controller, like the other test below.

Reply:
Since we never pass the controller to the second scrollable(footer), can we safely assume that it is always DataTable that uses the provided ScrollController?

Reply:
That is a great question! I imagine it is possible the user may want access to one controller or the other at some point. Does the issue specify which scrollable is needed for access? If so, it may require we have clearer parameters here that indicate which controller this is.

Reply:
Issue mentions that it needs controller for contents of PaginatedDataTable.  I assume that it is for main content.

Reply:
Hi @Piinks,  Should I expose both the controllers?

Reply:
Sure, I did ask and the author only needs the controller for DataTable.